README.md

```diff
@@ -241,3 +243,8 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 * Run `bundle install` then run `bundle exec pod install`
 * Open `RInAppMessaging.xcworkspace` in Xcode then build/run
 * To run the tests press key shortcut command-U
+
+## **Troubleshooting**
+
+* (For our internal users) Getting error 401 when calling ping/impression endpoint - check your app's scopes list if it contains all required entries OR stop putting accessToken in `IAMPreference` (breaks user targeting)
```

Comment:
Instead of this, lets say
> Rakuten developers experiencing problems should refer to the Troubleshooting Guide on the internal developer documentation portal.

so that we keep troubleshooting info in one place and can combine it with backend info.